apps/web/dynamic_filters/base.py

```diff
@@ -170,23 +170,61 @@ def apply_excludes(self, queryset, value, timezone=None) -> QuerySet:
 
 class StringColumnFilter(ColumnFilter):
     type: str = TYPE_STRING
+    or_columns: list[str] = Field(default_factory=list)
+
+    def _apply_with_lookup(self, queryset, lookup, value):
+        """Apply filter with optional OR logic across multiple columns.
+        queryset: The queryset to filter
+        lookup: The Django lookup to use (ex., 'icontains', 'istartswith'), or None for exact match
+        value: The value to filter by
+        """
+        if not self.or_columns:
```

Comment:
Suggestion:
If you rename `or_columns` to `columns`, update all places where `StringColumnFilter(colum="xyz")` to `StringColumnFilter(colums=["xyz"])` and remove the `if not self.or_columns:` checks, then it should also preserve the current behaviour while automatically applying the OR behaviour when there are multiple columns specified. This feels like an easier API to work with, since we only have to work with the `columns` field?

Thoughts?

Reply:
I'm not opposed to that change thought it touches a lot more code. I think it's worth getting this merged and tackling that change separately.